app/presenters/tree_builder_resource_pools.rb

```diff
@@ -3,7 +3,7 @@ class TreeBuilderResourcePools < TreeBuilderAlertProfileAssign
 
   def override(node, object)
     node.selectable = false
-    node.checkable = true
+    node.checkable = true if object.kind_of?(ResourcePool)
```

Comment:
Where is this coming from?

Reply:
If I remember correctly we tracked down a strange behavior and you let me put in my tree not in a new PR. But it's was in December so I am not remembering the exact circumstances

Reply:
Sounds legit to me :+1: